scripts/pipelines/azure-devops/build_pipeline_generator.sh

```diff
@@ -0,0 +1,122 @@
+while getopts n:l:d:b:p: flag
+do
+    case "${flag}" in
+        n) name=${OPTARG};;
+        l) language=${OPTARG};;
+        d) directory=${OPTARG};;
+        b) branch=${OPTARG};;
+        p) openPR=${OPTARG};;
+    esac
+done
+
+if test "$1" = "-h"
+then
+    echo ""
+    echo "Generates a build pipeline on Azure DevOps from a YAML template according to the project programming language or framework."
+    echo ""
+    echo "Arguments:"
+    echo "  -n    [Required] Name that will be set to the build pipeline."
+    echo "  -l    [Required] Language or framework of the project."
+    echo "  -d    [Required] Local directory of your project (the path should always be using '/' and not '\')."
+    echo "  -b               Name of the branch for which the Pull Request will be created."
+    echo "  -p               Open the Pull Request on the web browser if the merging fails. Accepted values: true, false."
+    exit
+fi
+
+white='\e[1;37m'
+green='\e[1;32m'
+red='\e[0;31m'
+
+# Argument check.
+if test -z "$name" || test -z "$language" || test -z "$directory"
+then
+    echo -e "${red}Missing parameters, all flags are mandatory."
+    echo -e "${red}Use -h flag to display help."
+    echo -e ${white}
+    exit
```

Comment:
```suggestion
    exit 2
```

Exit code 2 is usually used when user input is invalid. Anyway, it shouldn't be 0 (the default for success).